client/app/lib/providers/computecontroller.ui.coffee

```diff
@@ -182,11 +182,15 @@ module.exports = class ComputeControllerUI
 
 
       buttons['Advanced Mode'] =
+        title    : 'Advanced Mode'
         style    : 'solid medium advanced-mode-btn'
         type     : 'button'
         callback : ->
           form.toggleClass 'in-advanced-mode'
-          @toggleClass 'green'
+          if @buttonTitle is 'Advanced Mode'
```

Comment:
@sinan 
I think `Advanced Mode` button should be renamed instead of changing its class because it is still secondary action but not sure for `Basic Mode` title